README.md

```diff
@@ -19,66 +25,56 @@ All the files refers to the `SLUS-00067` version of the game.
+  * Run `SOURCE=src/st/wrp/6FD0.c make ctx.c` then `ASSEMBLY=asm/st/wrp/nonmatchings/6FD0/func_801873A0.s make decompile` to dump the decompiled code into ctx.c.m2c
+Replace the `INCLUDE_ASM(...);` you targeted with the content of `ctx.c.m2c` and tweak the code with `asm-differ`
+You will probably have some differences from your compiled code to the original; keep refactoring the code and move variables around until you have a 100% match.
 
 There are a few tricks to make the process more streamlined:
-* Use [decomp.me](https://decomp.me/new) with GCC 2.7.2 for PS1. Be aware that the repo is using GCC 2.6.x, so decomp.me will sometimes give a slightly different output
-* Use [decomp-permuter](https://github.com/simonlindholm/decomp-permuter) to [solve some mismatches](https://github.com/mkst/esa/wiki#decomp-permuter)
+* Use [decomp.me](https://decomp.me/) with GCC 2.7.2 for PS1. Be aware that the repo is using GCC 2.6.x, so decomp.me will sometimes give a slightly different output. 
+* the “context” section of decomp.me, is provided by the cmd `SOURCE=src/st/wrp/6FD0.c make ctx.c` as mentionned in the how to decompile.
+* Use [decomp-permuter](https://github.com/simonlindholm/decomp-permuter) to solve some mismatches
 * Use [this](https://github.com/mkst/sssv/wiki/Jump-Tables) and [this](https://github.com/pmret/papermario/wiki/GCC-2.8.1-Tips-and-Tricks) guide to understand how some compiler patterns work
 * Use the `#ifndef NON_MATCHING` if your code is logically equivalent but you cannot yet fully match it
 
-# Notes
 
-* I suspect that GCC 2.6.x / PSY-Q 3.4 have been used to originally compile `DRA.BIN`
-* `main.exe` uses PS-X libraries that might have been created with a different compiler and with `-O1` rather than `-O2`
+## Ressources:
+* list of ressource for sotn https://github.com/TalicZealot/SotN-Utilities (speedrun oriented, but very useful still). eg: 
+  * PS1’s CPU R3000 instruction [manual](https://cgi.cse.unsw.edu.au/~cs3231/doc/R3000.pdf) and [cheat sheet](https://vhouten.home.xs4all.nl/mipsel/r3000-isa.html)
+  * https://github.com/KernelEquinox/SNEER
+  * Debugging Emulator: https://github.com/grumpycoders/pcsx-redux
+  * Debugging Emulator with Register Breakpoints: https://www.romhacking.net/utilities/267/
+* beginner friendly MIPS video lectures [1](https://www.youtube.com/watch?v=PlavjNH_RRU&list=PLylNWPMX1lPlmEeeMdbEFQo20eHAJL8hx) and [2](https://www.youtube.com/watch?v=qzSdglU0SBc&list=PLylNWPMX1lPnipZzKdCWRj2-un5xvLLdK)
 
-## To do
 
+## To do:
 The project is very barebone at the moment and there is a massive room of improvement, mostly in the infrastructure:
-
 * Not all the zone overlays (`ST/{ZONE}/{ZONE}.BIN`) are disassembled
-* GNU AS might produce a slightly different register use compared to the original game. ASPSX might need to be used instead as a workaround but it is old, clunky and not yet integrated in the repo.
+* integrate ASPSX instead of GNU AS
```

Comment:
🔕 sorry for being picky, but let's start every phrase with a Capital letter for consistency!